config-raw.json

```diff
@@ -709,6 +709,11 @@
                                             "key": "emailfallback",
                                             "default_value": "false",
                                             "comment": "This option allows to look for a local account where the OIDC user's email match the Vikunja local email. Allowed value is either `true` or `false`. That option can be combined with `usernamefallback`.\nUse with caution, this can allow the 3rd party provider to connect to *any* local account and therefore potential account hijaking."
+                                        },
+                                        {
+                                            "key": "forceuserinfo",
+                                            "default_value": "false",
+                                            "comment": "This option forces the use of the OpenID Connect UserInfo endpoint to retrieve user information instead of relying on claims from the ID token. When set to `true`, user data (email, name, username) will always be obtained from the UserInfo endpoint even if the information is available in the token claims. This is useful for providers that don't include complete user information in their tokens or when you need the most up-to-date user data. Allowed value is either `true` or `false`."
```

Comment:
Once this is merged, please add a note about this to https://github.com/go-vikunja/website/blob/main/src/content/docs/setup/openid.mdoc as well.

Reply:
Sure, will updated after this PR is merged